scripts/background.js

```diff
@@ -138,7 +138,8 @@ async function snoozeInBackground(item, tab) {
 	var title = !isHref ? tab.title : (item.linkText ? item.linkText : item.selectionText);
 	var wakeUpTime = snoozeTime.valueOf();
 	var pinned = !isHref && tab.pinned ? tab.pinned : undefined;
-	var assembledTab = Object.assign(item, {url, title, pinned, startUp, wakeUpTime})
+	var cookieStoreId = tab.cookieStoreId;
```

Comment:
`cookieStoreId` property is present if browser is Firefox and the extension has "cookies" permission ([source](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/tabs/Tab#cookiestoreid)).